### PR TITLE
63 add readme

### PR DESCRIPTION
**Pull request by POPPININ**
Updated README.md with instructions on how to run our program. Instructions contain GIFs which provide visual instruction to the user.

### CODE REVIEW

**Review by Fard-Faru — DISMISSED** [review]
Looks Good

**Review by DanielLivshits — DISMISSED** [review]
Looks good with the new explanation ok the gifs!

**Review by missionStrawberry — CHANGES_REQUESTED** [review]
One minor typo but otherwise looks good!

File: README.md [hunk 1]
```diff
@@ -1,10 +1,44 @@
 # Project Template
 
-This repository contains code for our groups CSC207 project, a Task and Event Management application.
+This repository contains code for our groups CSC207 project, a Task and Event Management application. Through our program, a user can
+manage their weekly
+1. __Tasks__ which can be thought of as "to-do"s that don't have any timeframe attached to them. For example, 'Bake Cookies' is a task.
+2. __Events__ which have a specified start and end time, unlike tasks. For example, 'CSC207 Lecture from 1 to 2 PM' is an event.
 
-## Checklist For Your Project
+Our program allows the user to Create Tasks and Events, Modify Tasks and Events, Mark Tasks as complete or incomplete, Delete Tasks and Events and clear their weekly schedule in case the user wants to start from scratch.
```
[the remark this answers is not in the document]
Comment (DanielLivshits) [reply README.md:8]: Incomplete is the correct term.

[the remark this answers is not in the document]
Comment (missionStrawberry) [reply README.md:8]: for further clarification, uncompleted means something is not finished, and incomplete means something is not whole, or is missing parts. 
It should read "Mark Tasks as completed or uncompleted"

File: README.md [hunk 2]
```diff
@@ -1,10 +1,44 @@
...
+![Create Event](images/ModifyEvent.gif)
+
+### Modify Task
+
+![Create Event](images/ModifyTask.gif)
+
+### Delete Event
+
+![Create Event](images/CreateEvent.gif)
+
```
Comment (eatingcorn) [inline README.md:33]: Delete event is currently showing the create event gif

**Review by eatingcorn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (missionStrawberry) [reply README.md:8]: @DanielLivshits why "incomplete"

File: README.md (hunk 2 above)
Comment (missionStrawberry) [reply README.md:33]: Good catch Ricky

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (DanielLivshits) [reply README.md:8]: disregard, I was mistaken.

[the remark this answers is not in the document]
Comment (missionStrawberry) [reply README.md:8]: Sounds good.

File: README.md (hunk 2 above)
Comment (POPPININ) [reply README.md]: Thank you for catching that Ricky. Fixed it.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (POPPININ) [reply README.md]: Revised the sentence to "mark Tasks as completed or uncompleted".

File: README.md [hunk 3]
```diff
@@ -1,10 +1,44 @@
...
+![Create Event](images/DeleteTask.gif)
+
+### Complete Task
+
+![Create Event](images/CompleteTask.gif)
+
+### Clear All
+
+![Create All](images/ClearWeek.gif)
```
Comment (DanielLivshits) [inline README.md:44]: The alt text should be fixed to align with the correct name for each gif.

File: README.md [hunk 4]
```diff
@@ -1,10 +1,44 @@
 # Project Template
```
Comment (DanielLivshits) [inline README.md:1]: Should we change this, possibly to the title of our project (i.e. "Blue Guppies - Clean Calendar") or something along those lines?

**Review by DanielLivshits — CHANGES_REQUESTED** [review]
one necessary set of changes and one comment to be addressed for the README.

File: README.md (hunk 4 above)
Comment (POPPININ) [reply README.md]: Changed it to "Clean Calendar"

**Review by DanielLivshits — APPROVED** [review]
Looks Good!

**Review by Fard-Faru — DISMISSED** [review]

**Review by eatingcorn — DISMISSED** [review]
Thanks for addressing the changes, looks good to me.

**Review by missionStrawberry — DISMISSED** [review]
Almost there.

**Review by eatingcorn — APPROVED** [review]

**Review by scissorkhizer — APPROVED** [review]
Contains clear description of the program and its functionality and instructions on how to run it, gifs provided demonstrate the features well, good job!

**Review by Fard-Faru — APPROVED** [review]
Looks good

**Comment by DanielLivshits** [comment]
(Last change request was dismissed because it was resolved, but GitHub wasn't treating it as resolved for some reason.)

**Pull request merged**